itou/job_applications/migrations/0035_link_diagnoses.py

```diff
@@ -0,0 +1,83 @@
+from django.db import migrations
+
+# Use imports (application defined models) instead of `apps.get_model()`
+# (migration defined models). This can bite us later e.g. when running
+# tests in a fresh database that runs all migrations again and again).
+# However, we really need access to Managers, QuerySets and properties.
+from itou.eligibility.models import EligibilityDiagnosis
+from itou.job_applications.models import JobApplication
+
+
+def move_data_forward(apps, schema_editor):
+    """
+    Link eligibility diagnoses to job applications.
+    """
+
+    # I'm still asking myself if a data migration is really the best way.
```

Comment:
Tu te poses la question à cause des tests, c'est ça ? J'en comprends que cela risque de ralentir les tests ou de créer des soucis quand ils sont parallélisés.
Une solution serait de commenter le script (une fois passé en prod bien sûr) afin de conserver l'historique dans une migration qui, in fine, ne fait rien.

Reply:
Oui à cause des tests futurs et aussi du temps que prend cette migration (plusieurs minutes).

Je prend ta suggestion, je la trouve bien et elle permet de conserver l'historique de ce qu'on a fait.